vernac/topfmt.ml

```diff
@@ -168,6 +170,7 @@ let default_tag_map () = let open Terminal in [
     "message.error"    , make ~bold:true ~fg_color:`WHITE ~bg_color:`RED ()
   ; "message.warning"  , make ~bold:true ~fg_color:`WHITE ~bg_color:`YELLOW ()
   ; "message.debug"    , make ~bold:true ~fg_color:`WHITE ~bg_color:`MAGENTA ()
+  ; "message.prompt"   , make ~fg_color:`GREEN ()
```

Comment:
all the changes in this file can go

Reply:
I had changed the debugger prompt to be green in both CoqIDE and coqtop to make it easier for users to notice the prompt.  I hope we can keep that detail.